string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,162 @@
+/* exported titleCase */
+
+/** function capitlizeHyphen (hyphenWord) { //"in-depth"
+  const separatedWords = hyphenWord.split('-'); //['in', 'depth']
+  separatedWords[1] = separatedWords[1][0].toUpperCase() + separatedWords[1].slice(1); //['in', 'Depth']
+  return separatedWords.join('-'); //'in-Depth'
+} */
+
+function titleCase(title) {
+  const exceptions = ['a', 'an', 'the', 'as', 'at', 'by', 'for', 'in', 'of', 'on', 'per', 'to', 'and'];
+  const updatedTitle = [];
+  let str;
+  let specialCharPosition;
+  let specialChar;
+  if (title.includes(':')) {
+    specialCharPosition = title.indexOf(':');
+    specialChar = ':';
+    str = title.toLowerCase().split(':');
+    console.log('colon', str);
+  } else if (title.includes('-')) {
+    str = title.toLowerCase().split('-');
+  }
+  str = title.toLowerCase().split(' ');
+  console.log('wordSplit');
+  console.log(str);
+
+  for (let i = 0; i < str.length; i++) {
+    updatedTitle[i] = str[i][0].toUpperCase() + str[i].slice(1).toLowerCase();
+
+    if (exceptions.includes(str[i]) && i !== 0) {
+      updatedTitle[i] = str[i].toLowerCase();
+    } else if (str[i] === 'api') {
+      updatedTitle[i] = 'API';
+    } else if (str[i].includes('javascript')) {
+      updatedTitle[i] = 'JavaScript';
+    }
+  }
+  let joinedTitle = updatedTitle.join(' ');
+  if (specialChar) {
+    joinedTitle =
+      joinedTitle.substring(0, specialCharPosition) +
+      joinedTitle.substring(specialCharPosition);
```

Comment:
You'll need to find a way to capitalize the first word _after_ the colon in the string on line 42. 

You could isolate the first letter after the colon by using something like `specialCharPosition + 2` since the first word will always start at that index, capitalize it, and then add that newly modified string instead of the substring on line 42.